snews.php

```diff
@@ -945,7 +945,7 @@ function articles() {
 			if ($result = db() -> query($query_articles)) {
 				$link = '<a href="'._SITE;
 				while ($r = dbfetch($result)) {
-					$infoline = $r['displayinfo'] == 'YES' ? true : false;
```

Comment:
I agree with other syntax, but in this case I'll keep it (best practices - you know what I mean)